src/screens/Resources/ResourcesItemListScreen.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import {
+  ScrollView, View, TouchableOpacity, StyleSheet,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {
+  Text, ActivityIndicator,
+} from 'react-native-paper';
+import Collapsible from 'react-native-collapsible';
+import Accordion from 'react-native-collapsible/Accordion';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { NavigationEvents } from 'react-navigation';
+import { theme } from '../../style';
+import { nav } from './variables';
+
+export default function ResourcesItemListScreen(props) {
+  const { navigation } = props;
+  const snapshot = navigation.getParam('snapshot');
+  const [loading, setLoading] = useState(false);
+  const [activeSections, setActiveSections] = useState([]);
+  const [singleActive, setSingleActive] = useState(false);
+
+  function renderHeader(doc, _, isActive) {
+    return (
+      <View style={ItemListStyles.accordionHeader}>
+        <View style={ItemListStyles.accordionHeaderText}>
+          <Text style={ItemListStyles.resourceText}>
+            {doc.data().title}
```

Comment:
In case we have to do null checks, calling `doc.data()` here might make that hard to do. Can we call `doc.data()` in one place, store the result pass it to these components?